content/docs/user-guide/external-dependencies.md

```diff
@@ -1,23 +1,27 @@
 # External Dependencies
 
-There are cases when data is so large, or its processing is organized in a way
-such that you would like to avoid moving it out of its external/remote location.
-For example from a network attached storage (NAS), processing data on HDFS,
-running [Dask](https://dask.org/) via SSH, or having a script that streams data
+There are cases when data is so large, or its processing is organized in such a
+way, that its preferable to avoid moving it from its original location. For
+example data on a network attached storage (NAS), processing data on HDFS,
+running [Dask](https://dask.org/) via SSH, or for a script that streams data
 from S3 to process it.
 
-External <abbr>dependencies</abbr> and
+External dependencies and
 [external outputs](/doc/user-guide/managing-external-data) provide ways to track
 data outside of the <abbr>project</abbr>.
 
-## How it works
+## How external dependencies work
 
-You can specify external files or directories as dependencies for your pipeline
-stages. DVC will track changes in them and reflect this in the output of
-`dvc status`.
+External <abbr>dependencies</abbr> are considered part of the (extended) DVC
+project: DVC will track them, detecting when they change (triggering stage
+executions on `dvc repro`, for example).
 
-Currently, the following types (protocols) of external dependencies are
-supported:
+DVC can track files or directories on an external location as
```

Comment:
sounds like it repeats a lot of the previous paragraph?

Reply:
True 🤦. Updated in cfe6e07a.